srcpkgs/gamemode/template

```diff
@@ -0,0 +1,38 @@
+# Template file for 'gamemode'
+pkgname=gamemode
+version=1.5.1
+revision=1
+build_style=meson
+configure_args="-Dwith-systemd=false -Dwith-dbus-service-dir=/usr/share/dbus-1/services"
+hostmakedepends="git cmake meson ninja pkg-config"
+makedepends="inih-devel dbus-devel elogind-devel"
+depends="libgamemode polkit"
+short_desc="Optimise Linux system performance on demand"
+maintainer="Hyper-KVM <hyperkvmx86@gmail.com>"
+license="BSD-3-Clause"
+homepage="https://github.com/FeralInteractive/gamemode"
+distfiles="https://github.com/FeralInteractive/gamemode/archive/${version}.tar.gz"
+checksum="c510fd1391263c6fa05570de840865b1f0c8d573a7acbf488027327ae5a3bbde"
+subpackages="libgamemode libgamemode-devel"
```

Comment:
subpackages need only be set when order matters or when some need to be disabled conditionally.